src/runtime/impure/lfortran_intrinsics.c

```diff
@@ -225,6 +225,16 @@ LFORTRAN_API double _lfortran_dlog10(double x)
     return log10(x);
 }
 
+LFORTRAN_API float_complex_t _lfortran_clog10(float_complex_t x)
+{
+    return log10(x);
```

Comment:
This line fails to compile on Windows with the error:
```
D:\a\lpython\lpython\lpython-0.1.0-788-gcb5b5f606\src\runtime\impure\lfortran_intrinsics.c(230): error C2440: 'function': cannot convert from 'float_complex_t' to 'double'
```

Reply:
Typically those complex functions are prefixed with `c`.

Reply:
Also for single precision, they use the `f` suffix.

However, it seems that `log10` does not have a complex version? The `clog10` and `clog10f` don't seem to compile for me.

Reply:
So if this is needed in Python, you can compute it from Python using the complex log of base `e`.

Reply:
> you can compute it from Python using the complex log of base `e`.

Yeah, that should work too. Thanks!